git_machete/client.py

```diff
@@ -2271,36 +2273,35 @@ def create_github_pr(
 
         fork_point = self.fork_point(head, use_overrides=True, opt_no_detect_squash_merges=False)
         if not fork_point:
-            raise MacheteException(f"Could not find a fork-point for branch {head}.")
+            raise MacheteException(f"Could not find a fork-point for branch {bold(head)}.")
         commits: List[GitLogEntry] = self.__git.get_commits_between(fork_point, head)
         description_path = self.__git.get_main_git_subpath('info', 'description')
         description: str = utils.slurp_file_or_empty(description_path)
 
         ok_str = '<green><b>OK</b></green>'
-        print(fmt(f'Creating a {"draft " if opt_draft else ""}PR from `{head}` to `{base}`... '), end='', flush=True)
+        print(f'Creating a {"draft " if opt_draft else ""}PR from {bold(head)} to {bold(base)}... ', end='', flush=True)
         pr: GitHubPullRequest = create_pull_request(domain, org, repo, head=head, base=base, title=commits[0].subject,
                                                     description=description, draft=opt_draft)
         print(fmt(f'{ok_str}, see <b>{pr.html_url}</b>'))
```

Comment:
Actually, I'd change the URL to underline, more natural for URLs

Reply:
Both `head` and `base` are branch names, not URLs, i would stick to `bold`, wdyt?

Reply:
Yes, but in this line (`print(fmt(f'{ok_str}, see <b>{pr.html_url}</b>'))`) I'd change bold to underline